lib/resources/aws/aws_iam_policy.rb

```diff
@@ -141,6 +148,11 @@ def has_statement__normalize_criteria(criteria)
   end
 
   def has_statement__normalize_statements
+    # Some single-statement policies place their statement
+    # directly in policy['Statement'], rather than in an 
+    # Array within it.  See arn:aws:iam::aws:policy/AWSCertificateManagerReadOnly
+    # Thus, coerce to Array.
+    policy['Statement'] = [ policy['Statement'] ] if policy['Statement'].is_a? Hash
```

Comment:
`Array()` =^)

Reply:
No, this is where it doesn't work.  I tried it with Array here, and got the array-of-arrays-containing-keys-and-values and discussed above.